README.rst

```diff
@@ -913,6 +917,35 @@ Installing packages from the REPL of an embedded device works in this way:
 
 Further information on micropython-packaging can be found here: https://docs.micropython.org/en/latest/reference/packages.html
 
+Custom Health Check Endpoint
+----------------------------
+
+pypiserver provide a default health endpoint. ``/health`` It always returns
+``200 Ok`` if the service is up. Otherwise it means that the service is dead.
```

Comment:
```suggestion
``pypiserver`` provides a default health endpoint at ``/health``. It always returns
``200 Ok`` if the service is up. Otherwise, it means that the service is not responsive.
```

Some small wording suggestions :)